pysat/instruments/icon_euv.py

```diff
@@ -64,122 +64,7 @@ def load(fnames, tag=None, sat_id=None):
 
     """
     
```

Comment:
So... I tried to use this, but couldn't figure out the directory structure.  Is it not icon/euv/level_2 ?

Reply:
I fixed the icon_euv code to level_2. I'll fix it.

Reply:
Commit pushed.